streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -525,14 +525,13 @@ public void testStateGlobalThreadClose() throws Exception {
                 () -> streams.state() == KafkaStreams.State.PENDING_ERROR,
                 "Thread never stopped."
             );
-        } finally {
             streams.close();
```

Comment:
As above. (Maybe similar elsewhere; won't comment on it explicitly below)

Reply:
Ok, will do.

Reply:
@mjsax The line 532(original) wait for the streams state equal to **ERROR** after streams closed. 
Although this test case will pass if we remove `close()`, but it seems we might be better to retain `close()`?

Reply:
Sams comment as above. The Java compiler will insert a `close()` call and there is no need to explicitly call it any longer.